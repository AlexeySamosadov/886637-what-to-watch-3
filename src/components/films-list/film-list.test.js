import React from "react";
import renderer from "react-test-renderer";
import FilmsList from "./films-list.jsx";

const filmsData = [{
  name: `The Grand Budapest Hotel`,
  date: 2014,
  genre: `Drama`,
  src: `www.rr.i/fsfsdf`,
  id: `55`,
}];

it(`This is FilmList Unit Test`, ()=> {
  const tree = renderer.
  create(
      <FilmsList
        filmsData={filmsData}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
