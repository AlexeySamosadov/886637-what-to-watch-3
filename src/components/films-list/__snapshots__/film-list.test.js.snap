// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is FilmList Unit Test 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <video
      alt=""
      className=""
      height={175}
      poster="img/somePath"
      src="somePath"
      width={280}
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
